delete.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Cogctl.Actions.Relays.Delete do
+  use Cogctl.Action, "relays delete"
+
+  def option_spec do
+    []
+  end
+
+  def run(options, args, _config, %{token: nil}=endpoint) do
+    with_authentication(endpoint, &run(options, args, nil, &1))
+  end
+  def run(_options, [], _config, _endpoint), do: display_arguments_error
```

Comment:
Forgive me if I'm not understanding how the options bit works, but would it be clearer to include the relay name as a named argument and then requiring it with the `convert_to_params` bit?